loris/img.py

```diff
@@ -215,21 +215,21 @@ def __getitem__(self, image_request):
 		return fp
 
 	@staticmethod
-	def _link(to,frum):
+	def _link(to, frum):
 		link_dp = path.dirname(frum)
 		if not path.exists(link_dp):
 			makedirs(link_dp)
 		if path.lexists(frum): # shouldn't be the case, but helps debugging
 			unlink(frum)
-		symlink(to,frum)
-		logger.debug('Made symlink from %s to %s' % (to,frum))
+		symlink(to, frum)
+		logger.debug('Made symlink from %s to %s' % (frum, to))
 
-	def __setitem__(self, image_request, fp):
-		# The Image (fp) already exists, this simply makes a symlink in the
-		# cache to from the canonical syntax to the actual request path
+	def __setitem__(self, image_request, canonical_fp):
```

Comment:
Perhaps not so relevant to these particular changes, but it's a little confusing to me that setting the item in the cache just creates the symlink; where is the code that actually saves the canonical version and how does it relate to whether or not caching is configured in the webapp.py logic that calls this?